.github/workflows/build-py-dists.yml

```diff
@@ -8,8 +8,7 @@ jobs:
             fail-fast: false
             matrix:
                 os: [windows-latest, macos-latest]
-                python-version: [3.7, 3.8, 3.9, "3.10", "3.11", "3.12"]
-                python-arch: [x64]

```

Comment:
`python-arch` isn't needed here, and it caused an issue with the new ARM64 mac runners.

Reply:
I bet this is a holdover from when we were still doing x86 builds.  Makes sense to remove!